group-generators/helpers/data-providers/dune/provider.ts

```diff
@@ -0,0 +1,179 @@
+import { DuneErrorFactory } from './errors';
```

Comment:
In order to use the Data Provider in group generators you need to import it into the DataProvider index (group-generators/helpers/data-providers/index.ts)

- Import the DuneProvider
- Add the `DuneProvider` to the `dataProviders` object